Formula/das_tool.rb

```diff
@@ -0,0 +1,42 @@
+class DasTool < Formula
+  # cite Sieber_2018: "https://doi.org/10.1038/s41564-018-0171-1"
+  desc "Genomic binning refiner"
+  homepage "https://github.com/cmks/DAS_Tool"
+  url "https://github.com/cmks/DAS_Tool/archive/1.1.1.tar.gz"
+  sha256 "2a55f67b5331251d8fd5adea867cc341363fbf7fa7ed5c3ce9c7679d8039f03a"
+  head "https://github.com/cmks/DAS_Tool.git"
+
+  if OS.mac?
+    depends_on "coreutils"
+    depends_on "gettext"
```

Comment:
Is there run-time linkage against `gettext` on macOS? Check `brew linkage das_tool`.

Reply:
There is. Maybe this is the same dependency as `icu4c` on Linux?

```
❯ brew linkage das_tool
System libraries:
  /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation
  /usr/lib/libSystem.B.dylib
  /usr/lib/libc++.1.dylib
Homebrew libraries:
  /usr/local/opt/gettext/lib/libintl.8.dylib (gettext)
  /usr/local/opt/r/lib/R/lib/libR.dylib (r)
```

Reply:
Thanks for checking. Both `icu4c` and `gettext` are related to internationalization, but they are different packages. macOS includes `icu4c` with the OS but not `gettext`, and Linux is the opposite situation.